pkg/crinit/aggregated/aggregated.go

```diff
@@ -0,0 +1,399 @@
+	opts *aggregatedClusterRegistryOptions) (*rbacv1.ClusterRole, error) {
+
+	rule := rbacv1.PolicyRule{
+		Verbs:     clusterRoleVerbs,
+		APIGroups: clusterRoleAPIGroup,
+		Resources: clusterRoleResources,
+	}
+
+	cr := &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   clusterRole,
+			Labels: util.ComponentLabel,
+		},
+		Rules: []rbacv1.PolicyRule{rule},
+	}
+
+	if opts.DryRun {
+		return cr, nil
+	}
+
+	return clientset.RbacV1().ClusterRoles().Create(cr)
+}
+
+// createClusterRoleBinding creates the cluster role binding for the operations we will allow
+// on our resources
+func createClusterRoleBindings(cmdOut io.Writer, clientset client.Interface,
+	opts *aggregatedClusterRegistryOptions) (*rbacv1.ClusterRoleBinding,
+	*rbacv1.ClusterRoleBinding, error, error) {
+
+	// cluster role binding for clusterregistry.k8s.io:apiserver cluster role
+	crb1 := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   clusterRoleBindingAPIServer,
+			Labels: util.ComponentLabel,
+		},
+		Subjects: []rbacv1.Subject{
+			rbacv1.Subject{
+				Kind:      "ServiceAccount",
+				APIGroup:  "",
+				Name:      serviceAccount,
+				Namespace: opts.ClusterRegistryNamespace,
+			},
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     clusterRole,
+		},
+	}
+
+	// cluster role binding for system:auth-delegator cluster role
+	crb2 := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   clusterRoleBindingAuthDelegator,
+			Labels: util.ComponentLabel,
+		},
+		Subjects: []rbacv1.Subject{
+			rbacv1.Subject{
+				Kind:      "ServiceAccount",
+				APIGroup:  "",
+				Name:      serviceAccount,
+				Namespace: opts.ClusterRegistryNamespace,
+			},
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "system:auth-delegator",
+		},
+	}
+
+	if opts.DryRun {
+		return crb1, crb2, nil, nil
+	}
+
+	result1, err1 := clientset.RbacV1().ClusterRoleBindings().Create(crb1)
+	result2, err2 := clientset.RbacV1().ClusterRoleBindings().Create(crb2)
+	return result1, result2, err1, err2
+}
+
+// createAPIService creates the Kubernetes API Service to handle the cluster
+// registry objects
+func createAPIService(cmdOut io.Writer, clientset apiregclient.Interface,
+	opts *aggregatedClusterRegistryOptions,
+	ca *x509.Certificate) (*apiregv1beta1.APIService, error) {
+
+	glog.V(4).Infof("Creating Kubernetes API Service %v", clusterRegistryAPIService)
+
+	caBundle := cert.EncodeCertPEM(ca)
+
+	apiSvc, err := createAPIServiceObject(clientset, opts, caBundle)
+
+	if err != nil {
+		glog.V(4).Infof("Failed to create Kubernetes API Service %v: %v", apiSvc, err)
+		return nil, err
+	}
+
+	glog.V(4).Info("Successfully created Kubernetes API Service")
+	return apiSvc, nil
+}
+
+// createAPIServiceObject helper to create the cluster registry API Service
```

Comment:
nit: createAPIServiceObject creates and returns the cluster registry API Server object.

I don't think you need to specify that this is a helper.